colossalai/shardformer/modeling/falcon.py

```diff
@@ -0,0 +1,774 @@
+
```

Comment:
Please update support matrix in shard former's README.md and documentations

Reply:
updated